jbehave-support-core/src/main/java/org/jbehavesupport/core/rest/RestServiceHandler.java

```diff
@@ -125,6 +127,43 @@ public void sendRequest(String urlPath, HttpMethod requestMethod, ExamplesTable
         } catch (HttpClientErrorException | HttpServerErrorException e) {
             storeResponse(e);
         }
+        isSuccessful();
+    }
+
+    private void isSuccessful() {
+        if (testContext.keySet().containsAll(
+            Arrays.asList(REST_RESPONSE_CODE, REST_RESPONSE_HEADERS, REST_RESPONSE_JSON))) {
+            isSuccessful(HttpStatus.valueOf((Integer)testContext.get(REST_RESPONSE_CODE)),
+                testContext.get(REST_RESPONSE_HEADERS),
+                testContext.get(REST_RESPONSE_JSON));
+        } else {
+            log.warn("Test context doesn't have consistent REST response stored, skipping success handler");
+        }
+    }
+
+    /**
+     * Custom verifier for each response
+     * Method is internally called by API method {@link RestServiceHandler#sendRequest}.
+     */
+    public void isSuccessful(HttpStatus responseCode, HttpHeaders responseHeaders, Object response){
+
+    }
+
+    /**
+     * Each response that meant to be successful must match status
```

Comment:
that **IS** meant to be? Or is my english sense wrong?

Reply:
is it is.